packages/astro/src/cli/index.ts

```diff
@@ -127,8 +128,10 @@ async function runCommand(cmd: string, flags: yargs.Arguments) {
 		process.env.NODE_ENV = cmd === 'dev' ? 'development' : 'production';
 	}
 
+	const packageManager = (await whichPm(process.cwd())).name ?? 'npm';
+
 	const { notify } = await import('./telemetry/index.js');
-	await notify();
+	await notify(packageManager);
```

Comment:
Line 131 should be moved into the `notify()` function, we usually keep this file simple and only re-route commands to their main function.